### PR TITLE
Prettier config, standardized import-sort, dependency updates

### PR DESCRIPTION
**Pull request by fatso83**
Makes all formatting consistent

### CODE REVIEW

**Comment by ethanresnick** [comment]
I'm happy to add in prettier, but only with the default config (i.e., no `.prettierrc`). My take on prettier is that a big part of its value comes from helping teams/projects avoid pointless fights over style — but adding in a config file just reopens those fights (now you're fighting over what should go in the config).

**Comment by fatso83** [comment]
Sure, I have Prettier run on save in vim, and that resulted in lots of changes, so I tried adding a configuration to minimize the amount of changes (single vs double quotes, for instance).

I could change this to remove the configuration and add a Husky commit hook that runs Prettier instead? This would result in lots more changes though.

**Comment by fatso83** [comment]
Doing it.

**Comment by fatso83** [comment]
Did some smaller chores in the same PR (see commit list).

File: package.json [hunk 1]
```diff
@@ -15,6 +15,11 @@
     "example_4:watch": "npm run build && cd examples && npm run example_4:watch",
     "test": "npm run build && npm run mocha && npm run examples"
   },
+  "importSort": {
+    ".js": {
+      "style": "module"
```
Comment (ethanresnick) [inline package.json:20]: I'd never seen the import sort package before, or this module style, but I quite like its logic. May have to add this in other projects haha. Thanks!

File: src/buildMatchSnapshot.js [hunk 2]
```diff
@@ -73,10 +77,10 @@ const buildMatchSnapshot = (utils, parseArgs) => {
     //
     // Note: we only test the top-level data for cusotm serializers
     // (jest might look for them recursively).
-    const logCustomSerializerWarning = obj !== toMatch // we cloned
-      && getSerializers().some(it => it.test(obj)); // has custom serializers
+    const logCustomSerializerWarning =
+      obj !== toMatch && getSerializers().some(it => it.test(obj)); // we cloned // has custom serializers
```
Comment (ethanresnick) [inline src/buildMatchSnapshot.js]: Can you fix up the comment here (rewording as necessary)? It got a little weird with prettier

**Review by ethanresnick — COMMENTED** [review]
Looks great overall! See comments

File: package.json (hunk 1 above)
Comment (fatso83) [reply package.json:20]: Was on a project where different IDEs had different ideas on import sort order and formatting. This did away with those issues :-) It should match up with IntelliJ/Webstorm's sort order.

File: src/buildMatchSnapshot.js (hunk 2 above)
Comment (fatso83) [reply src/buildMatchSnapshot.js]: Missed that! Not totally sure about the fix:
`// was  cloned & has custom serializers`
?

Comment (ethanresnick) [reply src/buildMatchSnapshot.js]: Yeah, that works. Exact wording doesn't matter too much; just the `// ... // ....` was weird.

**Comment by fatso83** [comment]
Fixed comment

**Pull request merged**